renovate.json

```diff
@@ -14,5 +14,13 @@
   "suppressNotifications": [ "artifactErrors" ],
   "labels": [
     "dependency"
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["variables\\/defaults\\.yaml"],
+      "matchStrings": [
+        "datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\s.*?_version:(?<currentValue>.*)"
```

Comment:
I've not verified that this regex is correct.

Reply:
I've tested the regex. Also not 100% sure if it does, but as I've taken it from the Renovate [examples](https://docs.renovatebot.com/modules/manager/regex/#advanced-capture), it should.